### PR TITLE
refactor: Remove UBA functionality

### PR DESCRIPTION
**Pull request by pxrl**
This is complicating the v3 upgrade path. Remove it now; it can be restored post-migration when v2 support has been removed.

I've cast the net wide on this one and have attempted to flush as much UBA-related functionality as I can find. It could be the case that we don't want to remove this much - we can selectively restore anything that falls into that category.

nb. This is a huge diff, and in practice all of this functionality is unused in the current system. It feels somewhat violent to be removing this now, but it does also represent a risk to have so much deactivated code in the codebase.

### CODE REVIEW

File: src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts [hunk 1]
```diff
@@ -334,29 +326,8 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
 
       try {
         const parsedValue = parseJSONWithNumericString(args.value) as ParsedTokenConfig;
-
         const l1Token = args.key;
 
-        // For now use the presence of `uba` or `rateModel` to decide which configs to parse.
-        if (parsedValue?.uba !== undefined) {
```
Comment (pxrl) [inline src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts:341]: Note - assuming we also want to remove ConfigStore support for the UBA, we should be sure to also flush the config in the onchain update(s).

Comment (nicholaspai) [reply src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts:341]: agreed

**Review by nicholaspai — APPROVED** [review]

**Review by james-a-morris — APPROVED** [review]
I like this. There's a lot of tested-but-unproven code paths that are now removed - this feels like a net good.

File: src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts (hunk 1 above)
Comment (pxrl) [reply src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts:341]: Have recorded this in ACX-1846 :+1:

**Pull request merged**